source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -61,6 +61,8 @@ constexpr absl::string_view REDIRECT_LOGGED_IN = "oauth.logged_in";
 constexpr absl::string_view REDIRECT_FOR_CREDENTIALS = "oauth.missing_credentials";
 constexpr absl::string_view SIGN_OUT = "oauth.sign_out";
 
+const std::string& DEFAULT_AUTH_SCOPE = "user";
```

Comment:
I'd make this absl::string_view for consistency with the others unless you need this to be a std::string